src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -60,15 +57,19 @@ public void resetEncoders() {
   }
 
   public double getLeftWheelsRPM() {
-    return ((m_leftFront.getSelectedSensorVelocity() * Constants.DriveTrainConstants.k100msTo60sRatio) / 2048 + (m_leftRear.getSelectedSensorVelocity() * Constants.DriveTrainConstants.k100msTo60sRatio / 2048)) / 2 / 4;
+    return (rawSpeedToRPM(m_leftFront.getSelectedSensorVelocity()) + rawSpeedToRPM(m_leftRear.getSelectedSensorVelocity())) / 2 / 4;
   }
 
   public double getRightWheelsRPM() {
-    return ((m_rightFront.getSelectedSensorVelocity() * Constants.DriveTrainConstants.k100msTo60sRatio) / 2048 + (m_rightRear.getSelectedSensorVelocity() * Constants.DriveTrainConstants.k100msTo60sRatio / 2048)) / 2 / 4;
+    return (rawSpeedToRPM(m_rightFront.getSelectedSensorVelocity()) + rawSpeedToRPM(m_rightRear.getSelectedSensorVelocity())) / 2 / 4;
   }
 
   public double getAverageWheelsRPM() {
-    return (getLeftWheelsRPM() + getRightWheelsRPM()) / 4;
+    return (getLeftWheelsRPM() + getRightWheelsRPM()) / 2;
+  }
+
+  private double rawSpeedToRPM(double rawSpeed) {
+    return rawSpeed * DriveTrainConstants.k100msTo60sRatio / DriveTrainConstants.kEncoderResolution;
```

Comment:
Great, this line gives motor RPM, but would you ever want the motor RPM? Usually the wheel RPM is what counts (Multiplying by 4 as in the `get...RPM()` functions.

Reply:
Aren't we supposed to divide by 4? Because every 4 motor spins = 1 wheel spin

Reply:
Yes, my bad! You should divide by four, not multiply.
But the point I was trying to make still stands:
This function is supposed to translate raw motor speed to RPM, but whose RPM? The motor? Or the wheel?
Is there a case when you would rather use the RPM of the motor itself on a drivetrain?
If not, you can divide by four inside the function, making the function translate raw speed to wheel RPM.